cookies/collection_test.go

```diff
@@ -36,6 +36,9 @@ func TestUnitCollection(t *testing.T) {
 		So(cookie.Path, ShouldEqual, "/")
 		So(cookie.Domain, ShouldEqual, testDomain)
 		So(cookie.MaxAge, ShouldEqual, maxAgeBrowserSession)
+		if cookie.Secure == false {
+			Print("Ensure Environmental Variable DP_IS_LOCAL_ENV is set to TRUE, as per cookies.init at L39")
```

Comment:
I'm not a fan of having `L39` as that line number could easily change and this might not be updated